src/contrib/loggers/Akka.Logger.SemanticSerilog/StringFormat.cs

```diff
@@ -0,0 +1,11 @@
+namespace Akka.Logger.SemanticSerilog
+{
+    internal class StringFormat
```

Comment:
Maybe, the templates are however cached, so it is a one time translation, thus the parsing perf/resource usage is largely irrelevant